docs/modules/ROOT/pages/api/erc20.adoc

```diff
@@ -0,0 +1,431 @@
+
+Returns the remaining number of tokens that `spender` is allowed to spend on behalf of `owner` through <<transfer_from,transfer_from>>. This is zero by default.
+
+This value changes when <<approve,approve>> or <<transfer_from,transfer_from>> are called.
+
+[.contract-item]
+[[IERC20::transfer]]
+==== `[.contract-item-name]#++transfer++#++(recipient: ContractAddress, amount: u256) → bool++` [.item-kind]#external#
+
+Moves `amount` tokens from the caller's token balance to `to`.
+Returns `true` on success, reverts otherwise.
+
+Emits a <<Transfer,Transfer>> event.
+
+Requirements:
+
+- `recipient` cannot be the zero address.
+- The caller must have a balance of at least `amount`.
+
+[.contract-item]
+[[IERC20::transfer_from]]
+==== `[.contract-item-name]#++transfer_from++#++(sender: ContractAddress, recipient: ContractAddress, amount: u256) → bool++` [.item-kind]#external#
+
+Moves `amount` tokens from `sender` to `recipient` using the allowance mechanism.
+`amount` is then deducted from the caller's allowance.
+Returns `true` on success, reverts otherwise.
+
+Emits a <<Transfer,Transfer>> event.
+
+Requirements:
+
+- `sender` cannot be the zero address.
+- `recipient` cannot be the zero address.
+- `sender` must have a balance of at least `amount`.
+- The caller must have allowance for ``sender``'s tokens of at least `amount`.
+
+[.contract-item]
+[[IERC20::approve]]
+==== `[.contract-item-name]#++approve++#++(spender: ContractAddress, amount: u256) → bool++` [.item-kind]#external#
+
+Sets `amount` as the allowance of `spender` over the caller's tokens.
+Returns `true` on success, reverts otherwise.
+
+Emits an <<Approval,Approval>> event.
+
+Requirements:
+
+- `spender` cannot be the zero address.
+
+[.contract-item]
+[[IERC20::Transfer]]
+==== `[.contract-item-name]#++Transfer++#++(from: ContractAddress, to: ContractAddress, value: u256)++` [.item-kind]#event#
+
+Emitted when `value` tokens are moved from one address (`from`) to another (`to`).
+
+Note that `value` may be zero.
+
+[.contract-item]
+[[IERC20::Approval]]
+==== `[.contract-item-name]#++Approval++#++(owner: ContractAddress, spender: ContractAddress, value: u256)++` [.item-kind]#event#
+
+Emitted when the allowance of a `spender` for an `owner` is set.
+`value` is the new allowance.
+
+[.contract]
+[[ERC20]]
+=== `++ERC20++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/token/erc20/erc20.cairo[{github-icon},role=heading-link]
+
+[.hljs-theme-dark]
+```javascript
+use openzeppelin::token::erc20::ERC20;
+```
+
+Implementation of the <<IERC20,IERC20>> interface.
+
+[.contract-index]
+.External Functions
+--
+* xref:#increase_allowance[`++increase_allowance(self, spender, added_value)++`]
+* xref:#decrease_allowance[`++decrease_allowance(self, spender, subtracted_value)++`]
+
+[.contract-subindex-inherited]
+.IERC20
+
+* xref:#name[`++name(self)++`]
+* xref:#symbol[`++symbol(self)++`]
+* xref:#decimals[`++decimals(self)++`]
```

Comment:
This three methods are missing below.

Also, we should always scope the Ids, I think we should always scope the IDs, in case we are more modules.

```suggestion
* xref:#ERC20-name[`++name(self)++`]
* xref:#ERC20-symbol[`++symbol(self)++`]
* xref:#ERC20-decimals[`++decimals(self)++`]
```